bin/upgrade.bash

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+
+here="$(readlink -f "$(dirname "${0}")")"
+
+ROOT="$(readlink -f "${here}/../")"
+
+CK8S_STACK="$(basename "$0")"
+export CK8S_STACK
```

Comment:
```suggestion
export CK8S_STACK="$(basename "$0")"
```

Reply:
Could be wrong but I think shellcheck complains if you do this